src/modules/builder/nav-bar/components/TemplatesSlider.tsx

```diff
@@ -93,21 +91,21 @@ export const TemplateSlide = ({
 }) => {
   return (
     <li
-      className="splide__slide hover:cursor-pointer"
+      className="splide__slide hover:cursor-pointer px-12"
       onClick={() => {
         onChangeTemplate(id);
       }}
     >
       <div
         className={`h-[255px] w-[179px] rounded border relative ${
-          isActive ? 'bg-resume-50 border-resume-500' : 'border-[#a9a9a9]'
+          isActive ? 'bg-resume-50 border-resume-800' : 'border-resume-200'
         }`}
       >
         <Image src={thumbnail} alt={name} layout="fill" />
 
         {isActive && (
-          <div className="absolute top-3 right-3">
-            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'28px'} height={'20px'} />
+          <div className="absolute top-1 right-1 bg-white">
```

Comment:
while the image is loading in the slow network, white background is shown around tick icon

Reply:
Removed redundant `resume-bg-50` for selected template and added `rounded-full` to the tick image div. Now the bg will be white but rounded and will not be visible as the PNG of the image also loads simultaneously.